### ISSUE TITLE
Crash during Urobolus fight with Dart Solo

### ISSUE THREAD

**Issue by zack37** [Legend-of-Dragoon-Modding/Severed-Chains#1533]
I was fighting Urobolus with Dart solo (Lavitz and Shana were not in active party slots). I killed Urobolus and when the cut scene where Dart charges Urobolus, I got a crash with the following stack trace:
```
20:58:07.343 [Hardware legend.game.Scus94491BpeSegment_8002:326] INFO : Loading scripted TMD legend.game.types.CContainer@5339bbad (animation legend.game.types.TmdAnimationFile@1059754c)
20:58:07.392 [Hardware legend.game.scripting.ScriptManager:184] ERROR: Script 12 crashed!
20:58:07.392 [Hardware legend.game.scripting.ScriptManager:185] ERROR: File 5572/1[addr 0x75b0]
20:58:07.392 [Hardware legend.game.scripting.ScriptManager:186] ERROR: Parameters:
20:58:07.392 [Hardware legend.game.scripting.ScriptManager:187] ERROR:   Op param: 0xa7
20:58:07.392 [Hardware legend.game.scripting.ScriptManager:189] ERROR:   1: script[12].storage[27] -1
20:58:07.392 [Hardware legend.game.scripting.ScriptManager:189] ERROR:   2: script[12].storage[10] 14
20:58:07.393 [Hardware legend.game.scripting.ScriptManager:192] ERROR: Storage:
20:58:07.393 [Hardware legend.game.scripting.ScriptManager:194] ERROR:   1: 0xc
20:58:07.393 [Hardware legend.game.scripting.ScriptManager:194] ERROR:   2: 0xffffffff
20:58:07.393 [Hardware legend.game.scripting.ScriptManager:194] ERROR:   3: 0xffffffff
20:58:07.393 [Hardware legend.game.scripting.ScriptManager:194] ERROR:   4: 0xffffffff
20:58:07.393 [Hardware legend.game.scripting.ScriptManager:194] ERROR:   5: 0xffffffff
20:58:07.393 [Hardware legend.game.scripting.ScriptManager:194] ERROR:   6: 0xffffffff
20:58:07.393 [Hardware legend.game.scripting.ScriptManager:194] ERROR:   7: 0xffffffff
20:58:07.393 [Hardware legend.game.scripting.ScriptManager:194] ERROR:   8: 0x10000
20:58:07.393 [Hardware legend.game.scripting.ScriptManager:194] ERROR:   9: 0x0
20:58:07.393 [Hardware legend.game.scripting.ScriptManager:194] ERROR:   10: 0xffffffff
20:58:07.393 [Hardware legend.game.scripting.ScriptManager:194] ERROR:   11: 0xe
20:58:07.393 [Hardware legend.game.scripting.ScriptManager:194] ERROR:   12: 0xffffffff
20:58:07.393 [Hardware legend.game.scripting.ScriptManager:194] ERROR:   13: 0xffffffff
20:58:07.393 [Hardware legend.game.scripting.ScriptManager:194] ERROR:   14: 0xffffffff
20:58:07.393 [Hardware legend.game.scripting.ScriptManager:194] ERROR:   15: 0xffffffff
20:58:07.393 [Hardware legend.game.scripting.ScriptManager:194] ERROR:   16: 0xffffffff
20:58:07.393 [Hardware legend.game.scripting.ScriptManager:194] ERROR:   17: 0xffffffff
20:58:07.393 [Hardware legend.game.scripting.ScriptManager:194] ERROR:   18: 0xffffffff
20:58:07.393 [Hardware legend.game.scripting.ScriptManager:194] ERROR:   19: 0xffffffff
20:58:07.393 [Hardware legend.game.scripting.ScriptManager:194] ERROR:   20: 0xffffffff
20:58:07.393 [Hardware legend.game.scripting.ScriptManager:194] ERROR:   21: 0xffffffff
20:58:07.393 [Hardware legend.game.scripting.ScriptManager:194] ERROR:   22: 0xffffffff
20:58:07.393 [Hardware legend.game.scripting.ScriptManager:194] ERROR:   23: 0xffffffff
20:58:07.393 [Hardware legend.game.scripting.ScriptManager:194] ERROR:   24: 0xffffffff
20:58:07.393 [Hardware legend.game.scripting.ScriptManager:194] ERROR:   25: 0xffffffff
20:58:07.393 [Hardware legend.game.scripting.ScriptManager:194] ERROR:   26: 0xffffffff
20:58:07.393 [Hardware legend.game.scripting.ScriptManager:194] ERROR:   27: 0xffffffff
20:58:07.393 [Hardware legend.game.scripting.ScriptManager:194] ERROR:   28: 0xffffffff
20:58:07.393 [Hardware legend.game.scripting.ScriptManager:194] ERROR:   29: 0xffffffff
20:58:07.393 [Hardware legend.game.scripting.ScriptManager:194] ERROR:   30: 0xffffffff
20:58:07.393 [Hardware legend.game.scripting.ScriptManager:194] ERROR:   31: 0xffffffff
20:58:07.393 [Hardware legend.game.scripting.ScriptManager:194] ERROR:   32: 0xffffffff
20:58:07.393 [Hardware legend.game.scripting.ScriptManager:194] ERROR:   33: 0xffffffff
20:58:07.393 [Hardware legend.game.scripting.ScriptManager:197] ERROR: Call stack:
20:58:07.393 [Hardware legend.game.scripting.ScriptManager:203] ERROR:   1: 395
20:58:07.393 [Hardware legend.game.scripting.ScriptManager:203] ERROR:   2: 0
20:58:07.393 [Hardware legend.game.scripting.ScriptManager:203] ERROR:   3: 0
20:58:07.393 [Hardware legend.game.scripting.ScriptManager:203] ERROR:   4: 0
20:58:07.393 [Hardware legend.game.scripting.ScriptManager:203] ERROR:   5: 0
20:58:07.393 [Hardware legend.game.scripting.ScriptManager:203] ERROR:   6: 0
20:58:07.393 [Hardware legend.game.scripting.ScriptManager:203] ERROR:   7: 0
20:58:07.393 [Hardware legend.game.scripting.ScriptManager:203] ERROR:   8: 0
20:58:07.393 [Hardware legend.game.scripting.ScriptManager:203] ERROR:   9: 0
20:58:07.393 [Hardware legend.game.scripting.ScriptManager:203] ERROR:   10: 0
20:58:07.393 [Hardware legend.core.RenderEngine:1123] ERROR: Shutting down due to exception:
java.lang.RuntimeException: An error occurred while ticking script 12
	at legend.game.scripting.ScriptManager.executeScriptFrame(ScriptManager.java:206)
	at legend.game.scripting.ScriptManager.tick(ScriptManager.java:42)
	at legend.game.Scus94491BpeSegment.lambda$gameLoop$45(Scus94491BpeSegment.java:410)
	at legend.core.RenderEngine.lambda$init$17(RenderEngine.java:494)
	at legend.core.opengl.Window$Events.onDraw(Window.java:697)
	at legend.core.opengl.Window.tickFrame(Window.java:337)
	at legend.core.opengl.Action.run(Action.java:46)
	at legend.core.opengl.Action.tick(Action.java:32)
	at legend.core.opengl.Window.run(Window.java:314)
	at legend.core.RenderEngine.run(RenderEngine.java:1121)
	at legend.core.GameEngine.start(GameEngine.java:254)
	at legend.game.Main.main(Main.java:23)
Caused by: java.lang.ArrayIndexOutOfBoundsException: Index -1 out of bounds for length 72
	at legend.game.combat.Battle.scriptGetCharOrMonsterId(Battle.java:3446)
	at legend.game.scripting.ScriptState.scriptExecuteSubFunc(ScriptState.java:937)
	at legend.game.scripting.ScriptState.runOp(ScriptState.java:559)
	at legend.game.scripting.ScriptState.executeFrame(ScriptState.java:487)
	at legend.game.scripting.ScriptManager.executeScriptFrame(ScriptManager.java:180)
	... 11 more
```

**Comment by LordMonoxide** [comment]
Hey zack, we have a list of all known issues with nodart/unlocked parties here https://docs.google.com/spreadsheets/d/1z-hrFuQxHAxPFsilgRIB9bHpEhEsmMhlHwKXvvttybk/edit

Many of these are fixed but haven't been merged back to main yet. We'll have the fixes in in the near future.

**Comment by avionanx** [comment]
Closed by #1558 and #1572
